src/scripts/discord/DiscordPokemonCode.ts

```diff
@@ -0,0 +1,13 @@
+class DiscordPokemonCode extends DiscordCode {
+    constructor(pokemon: DataPokemon, price, description) {
+        const image = `assets/images/pokemon/${pokemon.id}.png`;
```

Comment:
If/When the image is used, I think we would want the shiny image when appropriate

Reply:
At the moment, the images are only used with the Discord bot.
Not sure they will be used in game at all, but wanted to keep all the data the same across both, so thought it should include an image parameter that the bot can use.
![image](https://user-images.githubusercontent.com/7288322/92327231-46239680-f0ac-11ea-890b-db01fab981a2.png)

Reply:
Makes sense, could we have a comment explaining this